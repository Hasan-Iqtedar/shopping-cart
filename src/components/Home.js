import React from "react";
import homeImage from "./../assets/home.png";
import "./../styles/main.css";

const Home = (props) => {
  return (
    <div id="home">
      <h1>
        Welcome to the <br></br> Store
      </h1>
      <img src={homeImage} alt="shop img" />
      {/* <div className="cart"></div> */}
    </div>
  );
};

export default Home;
