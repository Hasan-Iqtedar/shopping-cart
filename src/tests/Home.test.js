import { render } from "@testing-library/react";
import Home from "../components/Home";

test("Renders heading and image", () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
});
