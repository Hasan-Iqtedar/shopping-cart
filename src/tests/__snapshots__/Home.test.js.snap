// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders heading and image 1`] = `
<div>
  <div
    id="home"
  >
    <h1>
      Welcome to the 
      <br />
       Store
    </h1>
    <img
      alt="shop img"
      src="home.png"
    />
  </div>
</div>
`;
